### ISSUE TITLE
R-D performance

### ISSUE THREAD

**Issue by myaolee** [LiuLei95/PyTorch-Learned-Image-Compression-with-GMM-and-Attention#4]
Hi~Thanks for your sharing code. 
When i train this network with Flickr_2W datasets as your description, that is:
A、Crop the Flicker_2W images randomly and get an new dataset with 82G
B、Train the 4096_256 modle 
i encountered two problems as follow：
1) I cannot achieve the excellent R-D loss as you, So do you think what i should adjust to achieve the same R-D loss as you?
2) Sometimes, the R-D loss is nan. I do not  know why this happen.
3) In datasets.py, the methods of training data augmentation only includes RandomHorizontalFlip and RandomVerticalFlip? Did you use RandomResizedCrop and Normalize in your modle training?

**Comment by LiuLei95** [comment]
2. You can train the network used the pre-trained model continue, which is saved before R-D loss is nan. Sometimes, the R-D loss is nan, espectly in high bit-rate.
3. RandomResizeCrop is used in the  A、Crop the Flicker_2W images randomly and get an new dataset with 82G. And Normalize we don't used.

**Comment by Vincentlinzesheng** [comment]
Ok, I will try your advices. Again, Thanks for your reply.

**Comment by myaolee** [comment]
Ok, i will try your advices. Again, Thanks for your reply.
